internal/store/store.go

```
@@ -691,20 +718,63 @@ func (s *Store) MoveGroup(path, newParent string) error {
+	if parentID != "" {
+		parent, err := s.validParent(id, parentID)
+		if err != nil {
+			return err
+		}
+		group = parent.Group
+	}
+	tx, err := s.db.Begin()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
```

golangci-lint (2.12.2)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)